<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbScanOptions">
     <property name="title">
      <string>Scan options:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbStartingUrl">
        <property name="text">
         <string>Starting URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leStartingUrl">
        <property name="text">
         <string>http://www.google.com.ua/search?q=text</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbTextToFind">
        <property name="text">
         <string>Text to find:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leTextToFind">
        <property name="text">
         <string>text</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbDownloadingTasksLimit">
        <property name="text">
         <string>Simultaneous downloading tasks limit:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbDownloadingTasksLimit">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbScanningTasksLimit">
        <property name="text">
         <string>Simultaneous scanning tasks limit:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sbScanningTasksLimit">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sbMaximumTasksLimit">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbMaximumTasksLimit">
        <property name="text">
         <string>Maximum tasks limit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmToolbar">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnStartScan">
        <property name="text">
         <string>Start scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPauseResumeScan">
        <property name="text">
         <string>Pause scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStopScan">
        <property name="text">
         <string>Stop Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spRightToolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbScanResults">
     <property name="text">
      <string>Scan results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvScanResults">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnStartScan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPauseResumeScan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pauseResumeScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStopScan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startScan()</slot>
  <slot>pauseResumeScan()</slot>
  <slot>stopScan()</slot>
 </slots>
</ui>
